projects/fsastorefrontlib/src/cms-components/my-account/address-info/address-form/address-form.component.html

```diff
@@ -162,7 +162,11 @@
       </button>
     </div>
     <div class="col-md-12 col-lg-6 ml-auto">
-      <button class="btn btn-block btn-primary mb-3" type="submit">
+      <button
```

Comment:
Maybe even try to hide the button after submit, because disabled attribute can be manipulated through DOM.